compiler/luci/pass/include/luci/CircleOptimizer.h

```diff
@@ -34,6 +34,7 @@ class CircleOptimizer final
   {
     enum Algorithm
     {
+      FuseUnrolledGRUAsCustomGRU,
```

Comment:
plz add to bottom not on the top

Reply:
Actually, it is first draft, I will post new one with changes only in `compiler` part.